app/javascript/components/editor-config/index.js

```diff
@@ -1,7 +1,8 @@
 // An example setup, adapted from prosemirror-example-setup
 
-import { schema, titleSchema } from './schema'
+import { schema } from './schema'
 import setupPlugins from './plugins'
```

Comment:
maybe better to put this all in one file.

Reply:
Are you recommending that we put the contents of `schema.js` in `index.js`? I'm not sure that is a good idea because `schema` is also imported by `images.js` `keys.js` `menu.js` `rules.js` `title-keys.js` `title-menu.js`. 

Do you still think `schema.js` and `index.js` should be a single file?

Reply:
ok